### PR TITLE
ENG-1205 cluster scope

### PR DESCRIPTION
**Pull request by ampie**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-controller-coordinator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | b4daf27369e4aa79d2cfb939acf31368b2ef110f | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-k8s-controller-coordinator/PR-42/1.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 5ddf623f0281569d4b8b73e3f407ca246946b692 | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-k8s-controller-coordinator/PR-42/2.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr42**

File: charts/entando-k8s-controller-coordinator/values.yaml
Comment (Kerruba) [inline charts/entando-k8s-controller-coordinator/values.yaml:6]: Is this ok to implement a reference to a non entando repository?

**Review by Kerruba — APPROVED** [review]
LGTM

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-k8s-controller-coordinator/pull/42#pullrequestreview-493406882" title="Approved">Kerruba</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-controller-coordinator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/OWNERS)~~ [Kerruba]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr42**

**Pull request merged**
